examples/dreambooth/train_dreambooth_lora_flux.py

```diff
@@ -1908,6 +1911,9 @@ def get_sigmas(timesteps, n_dim=4, dtype=torch.float32):
                 ignore_patterns=["step_*", "epoch_*"],
             )
 
+        images = None
```

Comment:
great contribution thanks! I want to add these changes to the advanced flux script too, is this last part specifically helpful? since it happens after the weights  are already uploaded/pushed to the hub

Reply:
It actually depends on the limit of memory and the user's hardware, but it will clean some memory usages for sure